app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Utils/Networking.java

```diff
@@ -1,25 +1,37 @@
 package ithaca_transit.android.cornellappdev.com.ithaca_transit.Utils;
 
+import android.annotation.TargetApi;
+import android.os.Build;
+import android.support.annotation.RequiresApi;
+import android.util.Base64;
+
 import org.json.JSONException;
 import org.json.JSONObject;
 
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.io.OutputStream;
 import java.net.HttpURLConnection;
 import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.StringJoiner;
 
 public final class Networking {
```

Comment:
Is there plans to eventually remove this Networking class?

Reply:
@djr277 Jonvi, your thoughts?